lib/server.js

```diff
@@ -32,6 +32,10 @@ var Router = util.safeRequire('react-router');
 var match = Router.match;
 var RoutingContext = Router.RoutingContext;
 
+//for creating a URL
+var createMemoryHistory = require('history').createMemoryHistory;
+var createLocation = createMemoryHistory().createLocation;
```

Comment:
@chunkiat82 why do we need this?

https://github.com/rackt/react-router/blob/master/docs/guides/advanced/ServerRendering.md